### ISSUE TITLE
Uninstall scripts

### ISSUE THREAD

**Issue by Narga** [philplckthun/setup-simple-ipsec-l2tp-vpn#31]
Do you provide scripts to uninstall it? I'm using Arch Linux

**Comment by philplckthun** [comment]
No, unfortunately not yet. If it's urgent, you can backtrack the script and undo the changes.

**Comment by Narga** [comment]
I can't remove libreswan

**Comment by Narga** [comment]
I found libreswan in AUR, why do you built it from source

**Comment by philplckthun** [comment]
The AUR is something Arch Linux specific and won't be interoperable with other distributions, obviously. That's why libreswan is compiled and installed from source, which makes sense as the AUR just does the same.

In case of libreswan you should just remove the built folder, the service, installation folder and all symlinks to uninstall it.

It's admittedly a pain, as libreswan doesn't provide an uninstall script, but it's not too hard to do. That's why I'd do backups before installing libreswan or install it on a fresh, single-purpose machine.
